### PR TITLE
Optimize yq install

### PR DESCRIPTION
**Pull request by liangxia**
Use an image which already contains yq, so that we are no longer relying on curl, which is not stable enough.

### CODE REVIEW

File: ci-operator/step-registry/sdn/conf/sdn-conf-commands.sh [hunk 1]
```diff
@@ -3,12 +3,8 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-echo "## Install yq"
-curl -L https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq
-echo "   yq installed"
-
 /tmp/yq -i '.networking.networkType="OpenShiftSDN"' "${SHARED_DIR}/install-config.yaml"
```
Comment (jianlinliu) [inline ci-operator/step-registry/sdn/conf/sdn-conf-commands.sh:6]: The `yq` from cli-yq image is located in `/tmp` ?

Comment (liangxia) [reply ci-operator/step-registry/sdn/conf/sdn-conf-commands.sh:6]: Yes. As I want to make as less change as possible.

Comment (liangxia) [reply ci-operator/step-registry/sdn/conf/sdn-conf-commands.sh:6]: You can see several jobs rely on this step already passed

Comment (liangxia) [reply ci-operator/step-registry/sdn/conf/sdn-conf-commands.sh:6]: Added in https://github.com/openshift/release/pull/31245

Comment (liangxia) [reply ci-operator/step-registry/sdn/conf/sdn-conf-commands.sh:6]: To save your time, it is
```
FROM ocp/4.5:cli
RUN curl -L https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq
```

**Comment by zhaozhanqi** [comment]
@jluhrsen @trozet Could you help take a look?

**Comment by jluhrsen** [comment]
thank you @liangxia

**Pull request merged**